### ISSUE TITLE
bingmap not working on mydriving

### ISSUE THREAD

**Issue by ammarfassy** [Azure-Samples/MyDriving#588]
dears,
I deployed the mydriving app on my mobile. I have updated the logger.cs with the bing map API key. though its not showing me the map.

please advise,
thanks,
Ammar

**Comment by shivamsaviant** [comment]
Hey @ammarfassy,

I am facing the same problem with bing maps. It is loading on past trips screen as a thumbnail but not on the current trip and past trip details screen.
Did you find any success on that?

Thanks.

**Comment by ammarfassy** [comment]
yeah correct, exactly the same situation where i am. i had to give up this
application buddy. no one is supporting.

my thread as below on technet :

https://social.microsoft.com/Forums/en-US/azureiothub/thread/f58af6f5-fc23-423f-ac7a-652dd2f531a7/#0a0c8851-472c-4098-9654-b3b66e16efb0


On Wed, Dec 20, 2017 at 10:44 AM, Shivam Srivastav <notifications@github.com

**Comment by shivamsaviant** [comment]
@ammarfassy I found the solution for this.
Create a fresh google map api-key using google developer console.
Copy that api-key.
Replace google map api key in android manifest with your api-key.

And it worked for me. :)

**Comment by martinsmuts** [comment]
@shivamsaviant  Thank you for this advice. I found that the fresh Google Map API Key did not work straight away, but I had to create an "enabled" Maps API key. I followed the instructions by creating one on this page:

1. Navigate to the Google APIS: website: http://go.microsoft.com/fwlink/p/?LinkId=268303
2. Sign in with your Google account credentials, click Create Project, provide a Project name, then click Create.
3. Click on the menu item in the upper left corner, click "APIs & Services".
4. Click on the link "ENABLE APIs AND SERVICES"  in the upper middle part of the screen.
5. Click on the tab "Google Maps Android API".
6. Click "enable" - note you may have to select the project you created using a Google Maps Key. Here you can just use the Google Maps API key that has been enabled, and paste it in the Android Manifest file.
7. Rebuild Solution in Visual Studio.
8. Debug on your Android device.

Hope this helps someone :)
